modules/dataquery/jsx/hooks/usebreadcrumbs.tsx

```diff
@@ -28,13 +29,13 @@ function useBreadcrumbs(
                  */
                 onClick: (e: React.MouseEvent) => {
                     e.preventDefault();
-                    setActiveTab('Info');
+                    setActiveTab(Tabs.Info);
                 },
             },
         ];
-        if (activeTab == 'DefineFields'
-                || activeTab == 'DefineFilters'
-                || activeTab == 'ViewData') {
+        if (activeTab == Tabs.Fields
+                || activeTab == Tabs.Filters
+                || activeTab == Tabs.Data) {
```

Comment:
Is there a reason to not use strict equality here (and in a few other places) ?

Reply:
I guess? I'm not sure what it would change since the compiler should be enforcing the types, but sure.